Source/Core/Core/AchievementManager.h

```diff
@@ -309,22 +309,14 @@ class AchievementManager
 #include <string>
 
 #include "Common/CommonTypes.h"
-
-namespace ActionReplay
-{
-struct ARCode;
-}
+#include "Core/GeckoCode.h"
+#include "Core/ActionReplay.h"
```

Comment:
```suggestion
#include "Core/ActionReplay.h"
#include "Core/GeckoCode.h"
```
To satisfy the lint bot.